include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
Hmm it seems quite old: https://msdn.microsoft.com/en-us/library/e02fwk4s(v=vs.120).aspx